src/app-layout/__integ__/awsui-applayout.test.ts

```diff
@@ -51,7 +51,9 @@ describe.each(['classic', 'refresh', 'refresh-toolbar'] as Theme[])('%s', theme
         await expect(page.isDisplayed(wrapper.findToolsToggle().toSelector())).resolves.toBe(true);
       }
       await expect(page.isDisplayed(wrapper.findNavigationClose().toSelector())).resolves.toBe(true);
+      await expect(page.isExisting(wrapper.findOpenNavigationPanel().toSelector())).resolves.toBe(true);
```

Comment:
why do you use isExisting and not isDisplayed?

Reply:
Because `isExisting` is cheaper than `isDisplayed` and works for our case. This selector only exists when the panel is open.

* isExisting is [a direct webdriver API call](https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/src/commands/element/isExisting.ts#L58)
* isDisplayed however is [a whole lot of Javascript code](https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/src/scripts/isElementDisplayed.ts)

If you think symmetry is very important, I can change to `isDisplayed`, but I try to be frugal when possible

Reply:
No, it is fine like that - thanks for the explanation 👍